build.xml

```diff
@@ -243,23 +258,28 @@ TODO consider switching back to this at 9.8.0/9.9build
                 <attribute name="Main-Class" value="edu.csus.ecs.pc2.Starter"/>
             </manifest>
         </jar>
-    	
+
+    </target>
+
+	 <!-- build sub-projects unless BUILD_NO_SUBPROJECTS defined -->
+    <target name="buildSubProjects" unless="BUILD_NO_SUBPROJECTS"  description="build subj projects ">
```

Comment:
Change description to "build sub projects".

Reply:
pushed fix.